Updated README.md

densellp opened the pull request:
Added source for pthread_mutex_trylock, and added a comment to say that valgrind doesn't really work with threads.

densellp committed:
Updated README.md

Added source for pthread_mutex_trylock, and added a comment to say that valgrind doesn't really work with threads.

javawolfpack reviewed (changes requested):
I purposefully did not include trylock as the assignment should not use them.

densellp committed:
Updated README.md

Removed the resource for pthread_mutex_trylock, and slightly changed the message before the valgrind information.

densellp commented:
I made the requested changes, but now it only looks like I added 1 line of text
